docs/configuring-playbook-bridge-appservice-slack.md

```diff
@@ -1,6 +1,8 @@
 # Setting up Appservice Slack bridging (optional)
 
-**Note**: bridging to [Slack](https://slack.com) can also happen via the [mx-puppet-slack](configuring-playbook-bridge-mx-puppet-slack.md) and [mautrix-slack](configuring-playbook-bridge-mautrix-slack.md) bridges supported by the playbook.
+**Notes**:
+- Bridging to [Slack](https://slack.com) can also happen via the [mx-puppet-slack](configuring-playbook-bridge-mx-puppet-slack.md) and [mautrix-slack](configuring-playbook-bridge-mautrix-slack.md) bridges supported by the playbook.
+- Currently (as of November, 2024) this component is not available for new installation unless you have already created a classic Slack application (which the bridge makes use of in order to enable bridging between Slack and Matrix), because the creation of classic Slack applications has been discontinued since June 4 2024. The author of the bridge claims [here](https://github.com/matrix-org/matrix-appservice-slack/issues/789#issuecomment-2172947787) that he plans to support the modern Slack application and until then "the best (and only) option for new installations is to use the webhook bridging". The playbook supports the Webhook with [matrix-hookshot](configuring-playbook-bridge-hookshot.md). While the playbook supports `matrix-appservice-webhooks`, it has been deprecated in favor of matrix-hookshot.
```

Comment:
I think Hookshot's webhook payload was not very compatible with Slack's payload (nowadays).

When they say "webhook bridging", I don't know if they mean using a a webhook service (bridge) like Hookshot or rather a Slack bridge which relies on "webhook technology" to do its work. Because of this, I don't think referencing Hookshot and appservice-webhooks is of much use here.